File: python/euler_5.py
check = True
candidato = 1
divisores = list(range(1,20))
otroCheck = False
while check:
    for divisor in divisores:
        if candidato % divisor !=0:
            otroCheck = False
            break
        else:
            otroCheck = True
    if otroCheck == True:
        check = False
    candidato = candidato + 1

print(candidato -1) #off by one no se porque

File: julia/euler_5.jl
function euler5(n; check= true, otroCheck = false)
    candidato = 1
    divisores = 1:n
    while check
        for divisor in divisores
            if candidato % divisor !=0
                otroCheck = false
                break
            else
                otroCheck = true
            end
        end
        if otroCheck
            check = false
        end
        candidato +=1
    end
    return candidato - 1
end

